src/qt/governancelist.cpp

```diff
@@ -0,0 +1,331 @@
+
+    LOCK(cs_dip3list);
+
+    QString strToFilter;
+    ui->tableWidgetProposalsDIP3->setSortingEnabled(false);
+    ui->tableWidgetProposalsDIP3->clearContents();
+    ui->tableWidgetProposalsDIP3->setRowCount(0);
+
+    nTimeUpdatedDIP3 = GetTime();
+
+    for(int i = 0; i < governanceList.size(); i++){
+        CGovernanceObject *obj = (CGovernanceObject*)governanceList.at(i);
+
+        QString isActiveStr = "";
+        std::string err = "";
+
+        if(obj->IsValidLocally(err, false))
+            isActiveStr = "Yes";
+        else
+            isActiveStr = "No";
+
+        QDateTime creationTime = QDateTime::fromSecsSinceEpoch(obj->GetCreationTime());
+        QDateTime paymentStartTime = QDateTime::fromSecsSinceEpoch(obj->GetPaymentStartTime());
+        QDateTime paymentEndTime = QDateTime::fromSecsSinceEpoch(obj->GetPaymentEndTime());
+        int yesCount = obj->GetYesCount(VOTE_SIGNAL_FUNDING);
+        int noCount = obj->GetNoCount(VOTE_SIGNAL_FUNDING);
+        int abstainCount = obj->GetAbstainCount(VOTE_SIGNAL_FUNDING);
+        int absYesCount = obj->GetAbsoluteYesCount(VOTE_SIGNAL_FUNDING);
+        int paymentAmount = obj->GetPaymentAmount();
+        int cycles = obj->GetCycles();
+        int currentCycle = obj->GetCurrentCycle();
+
+        QString paymentAmountStr = "-";
+        QString cyclesStr = "-";
+        QString currentCycleStr = "-";
+        QString paymentStartStr = "-";
+        QString paymentEndStr = "-";
+        QString creationTimeStr = "-";
+
+        if(obj->GetObjectType() == GOVERNANCE_OBJECT_PROPOSAL){
+            paymentAmountStr = QString::number(paymentAmount);
+            cyclesStr = QString::number(cycles);
+            currentCycleStr = QString::number(currentCycle);
+            paymentStartStr = paymentStartTime.toString("dd.MM.yyyy HH.mm");
+            paymentEndStr = paymentEndTime.toString("dd.MM.yyyy HH.mm");
+            creationTimeStr = creationTime.toString("dd.MM.yyyy HH.mm");
+        }
+        QTableWidgetItem *statusItem = new QTableWidgetItem(isActiveStr);
+        QTableWidgetItem *amountItem = new QTableWidgetItem(paymentAmountStr);
+        QTableWidgetItem *cyclesItem = new QTableWidgetItem(cyclesStr);
+        QTableWidgetItem *currentCycleItem = new QTableWidgetItem(currentCycleStr);
+        QTableWidgetItem *absuluteYesCountItem = new QTableWidgetItem(QString::number(absYesCount));
+        QTableWidgetItem *yesCountItem = new QTableWidgetItem(QString::number(yesCount));
+        QTableWidgetItem *noCountItem = new QTableWidgetItem(QString::number(noCount));
+        QTableWidgetItem *abstainCountItem = new QTableWidgetItem(QString::number(abstainCount));
+        QTableWidgetItem *paymentStartItem = new QTableWidgetItem(paymentStartStr);
+        QTableWidgetItem *paymentEndItem = new QTableWidgetItem(paymentEndStr);
+        QTableWidgetItem *creationTimeItem = new QTableWidgetItem(creationTimeStr);
```

Comment:
There are a LOT of `new`s here, but only one `delete` added, I haven't looked to closely, but that seems quite problematic.